### PR TITLE
Add ngen roslyn script

### PR DESCRIPTION
**Pull request by pakrym**
@davidfowl 
ngen-ed roslyn loads successfully bringing first page generation time from 4 to 1.6 sec on my machine

### CODE REVIEW

File: HelloMvcFullAppDomain/ngen.ps1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+param($action, $exe)
+$appbase =  (Get-Item $exe).Directory
+
+C:\Windows\Microsoft.NET\Framework\v4.0.30319\ngen.exe $action /execonfig:$exe $appbase\Microsoft.CodeAnalysis.dll
```
Comment (davidfowl) [inline HelloMvcFullAppDomain/ngen.ps1]: $env:windir

File: HelloMvcFullAppDomain/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,50 @@
+*.doc  diff=astextplain
```
Comment (davidfowl) [inline HelloMvcFullAppDomain/.gitattributes]: This should be repository wide not in the project

File: HelloMvcFullAppDomain/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
+using System;
```
Comment (davidfowl) [inline HelloMvcFullAppDomain/Startup.cs]: I wouldn't duplicate the mvc sample. Just add the ngen script with some instructions.

Comment (pakrym) [inline HelloMvcFullAppDomain/Startup.cs]: Yep, except there are some changes to rsp searching code. Id prefer doing dependency context, implementing mvc services based on it, and then app domain on top

Comment (davidfowl) [inline HelloMvcFullAppDomain/Startup.cs]: Should be in another PR, just commit the ngen script and nothing more for this change.

**Comment by pakrym** [comment]
:up: :date: just script
